app/views/categories/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="w-full">
+  <% if notice.present? %>
+    <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
+  <% end %>
+
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-4xl">Categories</h1>
+    <%= link_to 'New category', new_category_path, class: "rounded-lg py-3 px-5 bg-blue-600 text-white block font-medium" %>
+  </div>
+
+  <div id="categories" class="min-w-full">
+    <%= render @categories %>
+  </div>
```

Comment:
- [ ] Kindly follow the design given for this project, the project can only be approved if all the requirements are met. Currently, your project categories page and other pages are different from the design.


![design](https://github.com/Reggeditt/tranzact/assets/53124548/f24bc6bd-032d-4c94-9cd9-1b2d6fb394a5)


![designs](https://github.com/Reggeditt/tranzact/assets/53124548/36be22c0-b5c2-4d16-bd96-f60e565f6f57)


![categorybtb](https://github.com/Reggeditt/tranzact/assets/53124548/8a2e0837-68c7-4770-91fe-522dcdaf4f94)

Reply:
Can be improved!